docs/guides/AI_CORE_DEPLOYMENT.md

```diff
@@ -0,0 +1,112 @@
+# AI Core Deployment
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Maven Dependencies](#maven-dependencies)
+- [Usage](#usage)
+  - [Create a Deployment](#create-a-deployment)
+  - [Delete a Deployment](#delete-a-deployment)
+
+## Introduction
+
+This guide provides examples on how to create and manage deployments in SAP AI Core using the SAP AI
+SDK for Java.
+
+## Prerequisites
+
+Before proceeding, ensure you have met the general requirements outlined in
+the [README.md](../../README.md#general-requirements). 
+
+Additionally, to carry out the examples in this guide, you need:
+
+- **An AI Core Configuration** created in SAP AI Core.
+    - <details>
+      <summary>Example configuration from the AI Core <code>/configuration</code> endpoint</summary>
+
+      ```json
+      {
+        "createdAt": "2024-07-03T12:44:08Z",
+        "executableId": "azure-openai",
+        "id": "12345-123-123-123-123456abcdefg",
+        "inputArtifactBindings": [],
+        "name": "gpt-35-turbo",
+        "parameterBindings": [
+          {
+            "key": "modelName",
+            "value": "gpt-35-turbo"
+          },
+          {
+            "key": "modelVersion",
+            "value": "latest"
+          }
+        ],
+        "scenarioId": "foundation-models"
+      }
+      ```
+      </details>
+
+## Maven dependencies
+
+Add the following dependencies to your `pom.xml` file:
+
+```xml
+
+<dependencies>
+    <dependency>
+        <groupId>com.sap.ai.sdk</groupId>
+        <artifactId>core</artifactId>
+        <version>${ai-sdk.version}</version>
+    </dependency>
+</dependencies>
+```
+
+See [an example pom in our Spring Boot application](../../sample-code/spring-app/pom.xml)
+
+## Usage
+
+## Create a Deployment
+
+Use the following code snippet to create a deployment in SAP AI Core:
+
+```java
+public AiDeploymentCreationResponse createDeployment() {
```

Comment:
(Minor)

I understand you copied it from the previous `README.md` but at this point I would like to ask to drop the method boilerplate.

And to extract a variable for `api = new DeploymentApi(getClient())`, this will slightly reduce the complexity.

And to extract a variable for `resourceGroup = "default"`, this will reduce ambiguousness.